sessions/introduction.qmd

```diff
@@ -21,4 +21,127 @@ the keyboard.
+-   Communicating the changes you've made to files in the project (as a
+    Git repository) by structuring commits as "atomic" changes and
+    writing commit messages following the Conventional Commit standard.
+-   Collaborating together by reviewing each other's contributions in
+    pull requests by making comments and suggestions.
+-   Accepting contributions by merging changes back into the main branch
+    of the project.
+
+### Why we want that {.unnumbered}
+
+Working well together on a common project isn't easy nor a thing we can
+naturally do. It requires a lot of skill, knowledge, and actually
+applying these practices often in real settings. Above all though, it
+requires *explicitly* agreeing and committing to working a certain with
+your team members. Agreeing to work better together can be easy. Knowing
+*how exactly* to work together is hard, *explicitly* agreeing on those
+is harder, and actually committing to those practices is the hardest.
+
+Too often, we just say "oh, it's just a small thing" or "I have to do
+this quickly" or "It's too much learning or change for me, I'll just do
+it my way". But, if we don't agree and actually commit to working better
+together, we'll end up with a mess of a project, with no one knowing
+what's going on, who's doing what, and what the current state of the
+project is. By following the same agreed-upon collaborative practices,
+you as a team can focus on the content, on understanding what others are
+contributing to the project, and providing relevant feedback directly to
+where it is needed.
+
+So, to help with *knowing how exactly to work together*, here are the
+reasons why we use the tools and practices we do based on our
+experiences and research into what are the most optimal and effective
+ways to collaborate together as a team:
+
+-   We use Git and GitHub because they are both incredibly popular tools
+    for collaboration.
+-   We use the GitHub flow (using Git branches and pull requests)
+    because it allows each team member to work on their own part of the
+    project without interfering with others' work and makes it easy to
+    get feedback and reviews from others when that work is ready to be
+    added back to the main content of the project.
+-   We use "atomic" commits, which means the commit contains a single
+    conceptual change (e.g. only changes a typo), so that it is easier
+    for the reviewers (our team members) to understand what we changed
+    and why.
+-   We use the Conventional Commits standard to write commit messages
+    because it ensures that messages are consistent between team members
+    and it more clearly communicates what was done in a commit.
+-   We use GitHub pull requests to be able to directly comment on and
+    give feedback to changes made by other team members so that review,
+    discussions, and decisions can be visible to everyone on the team.
+
+For more reasons on why we use these tools and practices, see the
```

Comment:
Good example of where drawing the line between a rostools `decisions` repo and a Seedcase `decisions` repo will be difficult (I know I'm the one who suggested it, but this is an example of why it might be difficult to split up, even though I would like it to be)